### ISSUE TITLE
Constant doesn't work

### ISSUE THREAD

**Issue by sharok** [JoshClose/CsvHelper#1732]
**Describe the bug**
The `Constant` method doesn't do anything and `MissingFieldException` is thrown.

**To Reproduce**
```
void Main()
{
	var config = new CsvConfiguration(CultureInfo.InvariantCulture)
	{
		MissingFieldFound = null // suppress the error
	};
	using (var stream = new MemoryStream())
	using (var writer = new StreamWriter(stream))
	using (var reader = new StreamReader(stream))
	using (var csv = new CsvReader(reader, config))
	{
		 csv.Context.RegisterClassMap<FooMap>();
		 
		writer.WriteLine("Id,Name");
		writer.WriteLine("1,one");
		writer.WriteLine("2,two");
		writer.Flush();
		stream.Position = 0;
		

		while (csv.Read()) {
			csv.GetRecord<Foo>().Dump();
		}
	}
	
}

private class Foo
{
	public string Id { get; set; }
	public string Name { get; set; }
	public string ConstValue { get; set; }
}

private class FooMap : ClassMap<Foo>
{
	public FooMap()
	{
		Map(m => m.Id).Index(0);
		Map(m => m.Name).Index(1);
		Map(m => m.ConstValue).Constant("HELLO");
	}
}
```
**Expected behavior**
The `Constant` should assign a value that specified. and the library should not throw  the `MissingFieldException`.

**Additional context**
Version 25.0.0.0

**Comment by JoshClose** [comment]
Creating a map for the property means you're mapping a property to a field in the csv file. If the field doesn't exist, it's going to throw that exception.

If you set `.Optional()` it won't throw the exception. It also doesn't set the constant, so I'll look into that. Setting a default does work though.

```cs
Map(m => m.ConstantValue).Optional().Default("HELLO");
```

**Comment by JoshClose** [comment]
I think setting `Constant` should negate the need to have a field attached, like you assumed. That's what I'm going to do.

**Comment by JoshClose** [comment]
This is fixed in 26.0.1 on NuGet. https://www.nuget.org/packages/CsvHelper/26.0.1

You can just set `Constant` like in your example. No `Optional` needed.

**Comment by sharok** [comment]
Great, thanks for the quick fix.
